shell/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "author": "Daniel Janz, Thorsten Schau & Samuel Andert (Basic Income Lab GmbH)",
   "license": "AGPL-3.0-or-later",
+  "description": "CirclesUbi Web App Frontend",
```

Comment:
Isn't it `CirclesUBI`? Or maybe even only the `CirclesUBI Indonesia web app`?
Nice review-feature you enabled there by the way :)